lib-php/website/reverse-only-search.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+require_once(CONST_LibDir.'/ParameterParser.php');
+
+$oParams = new Nominatim\ParameterParser();
+$sOutputFormat = $oParams->getSet('format', array('text', 'json'), 'text');
+
+if ($sOutputFormat == 'json') {
+    header('content-type: application/json; charset=UTF-8');
+    $aResponse = array(
+                  'status' => 404,
+                  'message' => 'Reverse-only import does not support forward searching.',
+                  'software_version' => CONST_NominatimVersion
+                 );
+    javascript_renderData($aResponse);
+} else {
+    header('HTTP/1.0 404 Not Found');
+    echo 'Error: Reverse-only import does not support forward searching.';
+}
```

Comment:
Instead of formatting this yourself, you could also use `set_exception_handler_by_format()` and then simply throw a Exception.

Reply:
Done.